lib/source-loader/src/abstract-syntax-tree/__snapshots__/inject-decorator.test.js.snap

```diff
@@ -393,25 +414,7 @@ storiesOf('ngrx|Store', module).addParameters({ storySource: { source: __STORY__
 
 exports[`inject-decorator positive calculates "adds" map 1`] = `
```

Comment:
checked this snapshot manually, we see lot of changes only because items in object were reordered because key was renamed and it orders it alphabetically